src/databasic/JsonPathTest.java

```diff
@@ -0,0 +1,49 @@
+package databasic;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class JsonPathTest {
+
+    @Test
+    public void canWriteAndGetJsonPath() {
+        JsonPath target = new JsonPath();
+        target.setJsonPath("number.value");
+        String result = target.getJsonPath();
+        Assertions.assertEquals("number.value", result);
+    }
+
+    @Test
+    public void givenNullString_throwsRuntimeException() {
+        JsonPath target = new JsonPath();
+
+        try {
```

Comment:
Now that we're more comfortable with exceptions, look online for a more elegant solution for verifying that exceptions are being thrown in JUnit tests. The solution I'm looking for uses an `@Annotation`.

Reply:
Junit 5 in this doc...